Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -96,9 +96,9 @@ public static class RelationshipsAuthorizationFilters
                         WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
                         @"{currentAlias}.StudentUSI IN (
                         SELECT {newAlias1}.StudentUSI 
-                        FROM " + "auth_StudentUSIToEducationOrganizationIdThroughEdOrgAssociation".GetFullNameForView() + @" {newAlias1} 
-                        WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
-                        (c, w, p, jt) => c.ApplyJoinFilter(w, p, "StudentUSIToEducationOrganizationIdThroughEdOrgAssociation", "StudentUSI", "SchoolId", jt),
+                        FROM " + "auth_SchoolIdToStudentUSIThroughEdOrgAssociation".GetFullNameForView() + @" {newAlias1} 
+                        WHERE {newAlias1}.SchoolId IN (:SchoolId))",
+                        (c, w, p, jt) => c.ApplyJoinFilter(w, p, "SchoolIdToStudentUSIThroughEdOrgAssociation", "StudentUSI", "StudentUSI", "SchoolId", jt),
```

Comment:
See note above about changes around ThroughEdOrgAssociation.

Reply:
I rebase with latest main to resolve this change